### PR TITLE
Missing "end" in example code

### PR DESCRIPTION
**Pull request by kwaping**
Indentation looks wrong in the diff but it's fine in the full-file view.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -218,7 +218,9 @@ http {
 
         -- errs is an array-like table
         if errs then
-            for i = 1, #errs do ngx.log(ngx.ERR, errs[i])
+            for i = 1, #errs do
+	        ngx.log(ngx.ERR, errs[i])
+	    end
```
Comment (p0pr0ck5) [inline README.md]: to be pedantic, mind if we align this `end` with the `for` above?

Comment (kwaping) [reply README.md]: I will double-check but I thought it was already aligned.

Comment (kwaping) [reply README.md]: Yeah, in the whole-file view mode it looks aligned: https://github.com/kwaping/lua-resty-waf/blob/77006f326aa4bb9406091a2d74435c7faa0d26a8/README.md

Comment (p0pr0ck5) [reply README.md]: Not particularly interested in whole-file view :) We can clearly see these are misaligned. tabs vs spaces perhaps?

**Comment by kwaping** [comment]
Sorry about that! It was my first time editing with the github.com built-in editor, instead of my usual IDE. I got it fixed, thank you for pressing me on that.

**Comment by p0pr0ck5** [comment]
Sorry for the delay, this is merged. Thank you!

**Pull request merged**